pelorus-operator/Makefile

```diff
@@ -3,7 +3,7 @@
 # To re-generate a bundle for another specific version without changing the standard setup, you can:
 # - use the VERSION as arg of the bundle target (e.g make bundle VERSION=0.0.2)
 # - use environment variables to overwrite this value (e.g export VERSION=0.0.2)
-VERSION ?= 0.0.5
+VERSION ?= 0.0.6-rc.1
```

Comment:
how this was generated?

For the next release we need to change how we invoke the script?

Reply:
understood

I would remove the changes made to `pelorus-operator` folder then, apply then in the next PR

Reply:
We can't as there is already check to ensure charts are in sync, so had to manually update those.

Reply:
https://github.com/dora-metrics/pelorus/pull/885/files#diff-63c82fa5dd1fbb3eef9b134e7f94495ad1b7e961e61efd5e27004c55b5abf70b

will have very soon follow up and the clean process, but don't want to make that huge PR...